src/eventable.js

```diff
@@ -36,8 +36,10 @@ function getEventableModule (notifyContext) {
   let listeners = {}
 
   function addListener (event, listener) {
-    listeners[event] = listeners[event] || []
-    listeners[event].unshift(listener)
+    event.split(' ').forEach(value => {
```

Comment:
To nitpick, I’d rename `event` to `events` because the string can potentially contain more than one event, separated by space.